otava/main.py

```diff
@@ -15,21 +15,21 @@
 # specific language governing permissions and limitations
 # under the License.
 
-import argparse
 import copy
 import logging
 import sys
 from dataclasses import dataclass
 from datetime import datetime, timedelta
 from typing import Dict, List, Optional
 
+import configargparse
```

Comment:
Actually it is common to `import configargparse as argparse` to avoid having to change the rest of the code.